app/api/pet_chat.py

```
@@ -1,12 +1,65 @@
+        if not pet_info:
+            raise HTTPException(status_code=404, detail="Pet not found")
+        return {"pet": pet_info}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching pet: {str(e)}")
+
+@llm_api.get("/owner/{owner_id}/pets")
+async def get_pets_by_owner_id(owner_id: int):
+    """MCP를 통해 소유자의 펫 목록 조회"""
+    try:
+        pets = await get_owner_pets(owner_id)
+        return {"pets": pets}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)